tools/dev-doctor/etcnfsconfcheck.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"gopkg.in/ini.v1"
+)
+
+// An etcdNFSConfCheck checks /etc/nfs.conf.
+type etcNFSConfCheck struct{}
+
+func (etcNFSConfCheck) id() string {
+	return "/etc/nfs.conf"
+}
+
+func (etcNFSConfCheck) run() (checkResult, string) {
+	data, err := ioutil.ReadFile("/etc/nfs.conf")
```

Comment:
What is this file and why is it needed? I don't have it, but my development and test VMs work fine with NFS.

Reply:
This file configures NFS. When you run `contrib/vagrant/start.sh` a message to ensure that nfsd is configured to use TCP if you have a firewall enabled is printed. That said, I've made the NFS check optional.

Reply:
Where do you see that message? Here's what I have:
```
$ K8S=1 NO_BUILD=1 NETNEXT=1 ./contrib/vagrant/start.sh 
# NFS enabled. don't forget to enable these ports on your host
# before starting the VMs in order to have nfs working
# iptables -I INPUT -p tcp -s 192.168.34.0/24 --dport 111 -j ACCEPT
# iptables -I INPUT -p tcp -s 192.168.34.0/24 --dport 2049 -j ACCEPT
# iptables -I INPUT -p tcp -s 192.168.34.0/24 --dport 20048 -j ACCEPT
# To use kubectl on the host, you need to add the following route:
# ip route add 192.168.33.11 via 192.168.34.11
Bringing machine 'k8s1+' up with 'virtualbox' provider...
==> k8s1+: Importing base box 'cilium/ubuntu-next'...
==> k8s1+: Matching MAC address for NAT networking...
==> k8s1+: Setting the name of the VM: cilium_k8s1_1603820305549_67317
==> k8s1+: Clearing any previously set network interfaces...
==> k8s1+: Preparing network interfaces based on configuration...
    k8s1+: Adapter 1: nat
    k8s1+: Adapter 2: intnet
    k8s1+: Adapter 3: hostonly
==> k8s1+: Forwarding ports...
    k8s1+: 6443 (guest) => 7443 (host) (adapter 1)
    k8s1+: 9081 (guest) => 9081 (host) (adapter 1)
    k8s1+: 2345 (guest) => 2345 (host) (adapter 1)
    k8s1+: 22 (guest) => 2222 (host) (adapter 1)
==> k8s1+: Running 'pre-boot' VM customizations...
==> k8s1+: Booting VM...
==> k8s1+: Waiting for machine to boot. This may take a few minutes...
    k8s1+: SSH address: 127.0.0.1:2222
    k8s1+: SSH username: vagrant
    k8s1+: SSH auth method: private key
    k8s1+: 
    k8s1+: Vagrant insecure key detected. Vagrant will automatically replace
    k8s1+: this with a newly generated keypair for better security.
    k8s1+: 
    k8s1+: Inserting generated public key within guest...
    k8s1+: Removing insecure key from the guest if it's present...
    k8s1+: Key inserted! Disconnecting and reconnecting using new SSH key...
==> k8s1+: Machine booted and ready!
==> k8s1+: Checking for guest additions in VM...
==> k8s1+: Setting hostname...
==> k8s1+: Configuring and enabling network interfaces...
==> k8s1+: Exporting NFS shared folders...
==> k8s1+: Preparing to edit /etc/exports. Administrator privileges will be required...
==> k8s1+: Mounting NFS shared folders...
==> k8s1+: Running provisioner: bootstrap (shell)...
```